lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.spec.ts

```diff
@@ -45,6 +45,8 @@ describe('ContentMetadataComponent', () => {
     let folderNode: Node;
     let tagService: TagService;
     let categoryService: CategoryService;
+    let classesApi: ClassesApi;
```

Comment:
I think you can define that classesApi inside beforeEach in 151st line and mark it as const